src/network/MessageContainer.cpp

```diff
@@ -14,10 +15,16 @@ namespace spy::network {
         MessageContainer::common_from_json(j, m);
     }
 
-    // TODO initialize creationdate
     MessageContainer::MessageContainer(messages::MessageTypeEnum messageType, util::UUID playerId) :
             playerId(playerId),
-            type(messageType) {}
+            type(messageType) {
+        time_t rawtime = time(nullptr);
+        struct tm creationDateTM = *gmtime(&rawtime);
+        creationDateTM.tm_hour = (creationDateTM.tm_hour + 1) % 24;
```

Comment:
Das geht kaputt sobald da eine Tages- (und damit auch Monats- Jahres-) Grenze überschritten wird.

Reply:
Stimmt...

Reply:
UTC+1 ist ein schlechter Standard. Habe es jetzt gefixt, indem ich die Umgebungsvariablen auf ein Land setzte, in dem immer UTC+1 ist...